README.md

```diff
@@ -1,152 +1,64 @@
+# To setup
 
-```
-npm install
-bower install
-```
+The backend is supported by Docker. If you do not have it already, you will need to install it.
+* For OS X, go [here](https://docs.docker.com/docker-for-mac/).
+* For Windows, go [here](https://docs.docker.com/docker-for-windows/).
+* For Ubuntu and other Linux distributions, install
+[docker](https://docs.docker.com/engine/installation/linux/ubuntulinux/) and
+[docker-compose](https://docs.docker.com/compose/install/).
+  To [avoid having to use sudo when you use the docker command](https://docs.docker.com/engine/installation/linux/ubuntulinux/#/create-a-docker-group),
+create a Unix group called docker and add users to it:
+  1. `sudo groupadd docker`
+  2. `sudo usermod -aG docker $USER`
 
+Once installed, start the Docker application (if on a Mac), then go to the project directory and run:
 
-You might also want to install Devtools for (React)[https://facebook.github.io/react/blog/2015/09/02/new-react-developer-tools.html] and (Redux)[https://github.com/zalmoxisus/redux-devtools-extension].
+1. `docker-compose up -d`
+2. `./init_docker.sh`
+3. `npm install`
+4. `bower install`
 
-####To develop
+You will only need to run the above commands once. Those will do the preliminary setup for the application by installing the dependencies and seeding the Docker containers to setup the database.
 
-Run `npm run dev` from the project dir to build and serve the development app.
+Use `docker-compose stop` to stop the containers or `docker-compose down` to both stop and remove the containers.
 
-####To deploy
+You might also want to install Devtools for [React](https://facebook.github.io/react/blog/2015/09/02/new-react-developer-tools.html). For Redux, you can install the [Google Chrome extension](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd) or use the other methods described in the README [here](https://github.com/zalmoxisus/redux-devtools-extension).
 
-Run `npm run deploy` and the output files will be written to the `dist` fodler.
+**Note:** If you are setting up Docker a second time, you may want to first remove the previous database containers with `docker-compose rm db`.
 
+# To develop
 
-## Text Thresher Backend
+In the project directory, run `docker-compose start` and `npm run dev`.
 
-The backend for Text Thresher, a tool for decomposing and annotating articles
-using the crowd. Under development for the
-[Deciding Force Project](http://www.decidingforce.org/).
+To view a browsable interface for the queries, navigate to `localhost:5000/api`. Another browsable interface is available [on Heroku](http://text-thresher.herokuapp.com/api/), but is not fully up-to-date.
 
-The latest version of the backend is running
-[on heroku](http://text-thresher.herokuapp.com/api). The API is browsable.
+**Note:** If you encounter an error that the module `text-highlighter/src/TextHighlighter` cannot be found, you will need to update brew by running `brew update`. 
```

Comment:
Please remove the README changes from this branch.
The commit this branch diverged from master is
`a8573af Merge pull request #2 from Goodly/loading-screen`
So to reverse:
```bash
git checkout highlighter-module
git checkout a8573af -- README.md 
git commit -m "Revert README.md changes."
```